addons/mail/static/src/discuss/call/common/call_menu.xml

```diff
@@ -3,17 +3,43 @@
 
     <t t-name="discuss.CallMenu">
         <div class="dropdown" t-attf-class="{{ className }}" t-ref="root">
-            <button t-if="rtc.channel" class="user-select-none dropdown-toggle o-no-caret o-dropdown--narrow" t-att-title="buttonTitle" role="button" t-on-click="rtc.channel.open">
-                <div class="o-discuss-CallMenu-buttonContent d-flex align-items-center o-gap-0_5 rounded-3 opacity-75 opacity-100-hover px-1" t-att-class="{ 'o-is-talking': rtc.selfSession?.isActuallyTalking, 'o-isOdooCommunity': !isEnterprise }">
-                    <span class="position-relative small bg-transparent">
-                        <i class="me-2 fa fa-fw" t-att-class="icon" />
-                        <small class="d-flex position-absolute top-0 end-0 smaller bg-inherit">
-                            <i class="o-discuss-CallMenu-animation fa fa-volume-up o-discuss-inCallIconColor bg-inherit" t-att-class="{ 'o-isOdooCommunity': !isEnterprise }"/>
-                        </small>
-                    </span>
-                    <span class="text-truncate fw-bold pe-1" t-esc="rtc.channel.displayName"/>
-                </div>
-            </button>
+            <div t-if="rtc.channel"
+                class="o-discuss-CallMenu-buttonContent d-flex align-items-center rounded-pill overflow-hidden"
+                t-att-class="{ 'o-is-talking': rtc.selfSession?.isActuallyTalking, 'o-isOdooCommunity': !isEnterprise }"
+            >
+                <button class="btn rounded-0 border-0 p-1 pe-2 gap-1 d-flex align-items-center h-100 flex-grow-1"
+                    title="Open Channel"
+                    t-on-click="rtc.channel.open">
+                    <div
+                        class="d-flex flex-shrink-0 align-items-center justify-content-center rounded-circle bg-primary text-white"
+                        style="width: 16px; height: 16px;">
+                        <img class="rounded-circle"
+                            t-att-src="rtc.channel.parent_channel_id?.avatarUrl ?? rtc.channel.avatarUrl"/>
+                    </div>
+                    <span class="text-truncate fw-bold text-dark lh-1"
+                        style="font-size: 13px;"
+                        t-esc="rtc.channel.displayName"/>
```

Comment:
Do we actually need the name of conversation? Honestly I would drop it, and keep the `volume-up` icon instead that is used to tell the conversation has an ongoing call, like in discuss sidebar:
<img width="301" height="174" alt="Image" src="https://github.com/user-attachments/assets/38c806dd-ed2f-4530-8493-8e0d4a047356" />
<img width="57" height="64" alt="Image" src="https://github.com/user-attachments/assets/a686632a-5707-4a7e-9695-2e1a38950914" />
Not showing the name would actually have the visual match somewhat the compact mode, which is good.

The systray horizontal space is precious, so not having the name is good. I think the `volume-up` icon is important to feel natural that this items maps to discuss calls.